packages/ffe-datepicker-react/src/calendar/MonthPicker.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { func, string } from 'prop-types';
+
+export default function MonthPicker(props) {
+    const { previousMonthHandler, nextMonthHandler, month } = props;
+
+    function left(evt) {
+        previousMonthHandler(evt);
+    }
+
+    function right(evt) {
+        nextMonthHandler(evt);
+    }
+
+    return (
+        <div className="ffe-calendar__monthpicker">
+            <div className="ffe-calendar__monthpicker-center">{month}</div>
+            <div
+                role="none"
+                onClick={left}
```

Comment:
Hvordan skall dette virka før tastaturbrukare og folk med screen readers?

Reply:
As I wrote [earlier](https://github.com/SpareBank1/designsystem/pull/777/files#r352883654).

“I did not think about screen readers. I suggest using css-element-queries instead if this design with breakpoints is to be implemented”.